src/test/java/emissary/util/PayloadUtilTest.java

```diff
@@ -243,14 +243,15 @@ void testXmlSerizliaztion() {
     void testIsValidForm() {
         // Check that all expected valid characters are valid
         String alphaLow = "abcdefghijklmnopqrstuvwxyz";
-        assertTrue(PayloadUtil.isValidForm(alphaLow), "Lower case alpha characters are not considered valid");
-        assertTrue(PayloadUtil.isValidForm(alphaLow.toUpperCase()), "Upper case alpha characters are not considered valid");
-        assertTrue(PayloadUtil.isValidForm("0123456789"), "Numeric characters are not considered valid");
-        assertTrue(PayloadUtil.isValidForm("-_"), "Dash and underscore aren't considered valid");
-        assertTrue(PayloadUtil.isValidForm("formName-(suffixInParens)"), "Parentheses aren't considered valid");
-        assertTrue(PayloadUtil.isValidForm("formName-(application/xml)"), "Slash aren't considered valid");
+        assertTrue(PayloadUtil.isValidForm(alphaLow), "Lower case alpha characters are valid");
```

Comment:
I think we should alter each messages in this test to more clearly indicate what the expected result is. The original and proposed messages are ambiguous regarding whether they're stating desired vs observed behavior, but we can take away the ambiguity:
```suggestion
        assertTrue(PayloadUtil.isValidForm(alphaLow), "Lower case alpha characters are expected to be valid");
```
or 
```suggestion
        assertTrue(PayloadUtil.isValidForm(alphaLow), "Expected lower case alpha characters to be valid");
```